fighthealthinsurance/ml/ml_router.py

```
@@ -68,25 +68,37 @@ def entity_extract_backends(self, use_external) -> list[RemoteModelLike]:
         else:
             return self.internal_models_by_cost
 
-    def summarize(self, text: Optional[str], abstract: Optional[str]) -> Optional[str]:
+    async def summarize(
+        self, text: Optional[str], abstract: Optional[str] = None
+    ) -> Optional[str]:
         models: list[RemoteModelLike] = []
```

Ruff (0.8.2)

`RemoteModelLike` may be undefined, or defined from star imports

(F405)